_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- Editors should not disclose any information about a manuscript under consideration to anyone other than those from whom professional advice is sought. After a decision has been made about a manuscript, but before publication, the editor may disclose or publish manuscript titles and authors’ names of papers that have been accepted for publication, but no more than that unless the author’s permission has been obtained. If a decision has been made to reject a manuscript for ethical violations, the editor may disclose the manuscript title and authors' names to other journal editors.
+- An editor should respect the intellectual independence of authors.
+- Editorial responsibility and authority for any manuscript authored by an editor and submitted to the editor’s journal should be delegated to some other qualified person, such as another editor of that journal / member of its Editorial Advisory Board. Editors should also delegate manuscripts submitted by recent graduates, or by colleagues at the same institution, to some other qualified editor. A collaboration between the author and editor, which is separate from the paper under review, or may may not represent a conflict requiring manuscript delegation. However, an editor should not be responsible for a paper which would directly further their own research program; for example, an editor should not manage a paper that will serve as preliminary results for a grant or grant renewal that the editor would financially benefit from, even if they were not involved in writing the grant.  Editors should review the LiveCoMS bylaws for more information on what constitutes a conflict. 
+- Unpublished information, arguments, or interpretations disclosed in a submitted manuscript should not be used in an editor’s own research except with the consent of the author. However, if such information indicates that some of the editor’s own research is unlikely to be profitable, the editor could ethically discontinue the work. When a manuscript is so closely related to the current or past research of an editor as to create a conflict of interest, the editor should arrange for some other qualified editor to take editorial responsibility for that manuscript. In some cases, it may be appropriate to tell an author about the editor’s research and plans in that area.
+- If an editor is presented with convincing evidence that the main substance or conclusions of a report published in an editor’s journal are erroneous, the editor should facilitate publication of an appropriate report pointing out the error.  This error should be corrected in the current GitHub version of the article, and a note may be placed on the original article indicating this correction. 
```

Comment:
I don't understand what situation this bullet point has in mind. Is it "Someone comes along proving that paper X is wrong, so the editor finds different authors to write a different paper showing that X is wrong?" Or is it "someone comes along proving that paper X is wrong, so the editor works with the authors of paper X to get the paper corrected or the error pointed out"? Or neitehr or both?

Reply:
Maybe it's just that they should ask the authors to correct it on the GitHub repository, and any future revisions would be accepted conditional on the concerns being resolved?

Reply:
Added text to that effect.